Dockerfile

```diff
@@ -17,13 +17,18 @@ RUN git clone https://www.github.com/docker/docker.github.io temp; \
 COPY . allv
 
 ## Branch to pull from, per ref doc
+## To get master from svn the svn branch needs to be 'trunk'. To get a branch from svn it needs to be 'branches/branchname'
+ENV ENGINE_SVN_BRANCH="trunk"
 ENV ENGINE_BRANCH="master"
+ENV DISTRIBUTION_SVN_BRANCH="branches/release/2.5"
 ENV DISTRIBUTION_BRANCH="release/2.5"
 
 # The statements below pull reference docs from upstream locations,
 # then build the whole site to static HTML using Jekyll
 
-RUN svn co https://github.com/docker/docker/branches/$ENGINE_BRANCH/docs/extend allv/engine/extend \
+RUN svn --non-interactive --trust-server-cert co https://github.com/docker/docker/$ENGINE_SVN_BRANCH/docs/ tmp_engine \
+ && mv tmp_engine/extend/* allv/engine/ \
```

Comment:
I think you may need to have a `mkdir -p allv/engine/extend/` and `mv tmp_engine/extend/* allv/engine/extend/`